package.json

```diff
@@ -101,11 +101,14 @@
     "pify": "3.0.0",
     "protobufjs": "6.8.6",
     "proxy-agent": "3.0.3",
+    "qrcode": "^1.4.4",
     "react": "16.8.3",
     "react-contextmenu": "2.11.0",
     "react-dom": "16.8.3",
     "react-portal": "^4.2.0",
+    "react-qrcode": "^0.2.0",
     "react-redux": "6.0.1",
+    "react-spinners": "^0.7.2",
```

Comment:
is the "react-spinners" in use at the end? otherwise you should remove the dependecy and update the yarn.lock file to